compiler/optimizer/RedundantAsyncCheckRemoval.cpp

```diff
@@ -1310,6 +1310,8 @@ void TR_RedundantAsyncCheckRemoval::getNearestAncestors(TR_StructureSubGraphNode
 
 void TR_RedundantAsyncCheckRemoval::markAncestors(TR_StructureSubGraphNode *node, TR_StructureSubGraphNode *entry)
    {
+   return;  // Disable it for Spark performance
```

Comment:
This doesn't seem like a good way of disabling something. I also think a bigger comment here is warranted as to why this is disabled and there should be a way to re-enable the feature in case this has undesirable side-effects elsewhere. Perhaps an envvar could be better here?